### ISSUE TITLE
Main website doesn't describe .puppet-lint.rc file

### ISSUE THREAD

**Issue by sparr** [rodjek/puppet-lint#770]
The README at https://github.com/rodjek/puppet-lint describes how to use .puppet-lint.rc but there is no mention of this functionality on http://puppet-lint.com/

**Comment by rodjek** [comment]
Yeah, the website is due for an overhaul. I'll make sure that this gets added to the new website.

**Comment by abuxton** [comment]
Any word on this would be good to link to from PDK doco or help flag too.
